home/web/client/index.html

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Parity Home</title>
     <script src="./inject.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
@tomusdrw In this case, there is actually only the "height:100%" style targeting #home, i.e. mountpoint - this is the one place where an inline style actually suits the usecase better.